fxgl-core/src/main/kotlin/com/almasb/fxgl/animation/Animation.kt

```diff
@@ -66,6 +66,23 @@ abstract class Animation<T>(
         }
     }
 
+    fun playReverse(){
+        if (!isAnimating) {
+            isReverse = true
+            play()
+        }
+    }
+
+    /**
+     * Plays Animation from current position in the direction indicated by rate.
+     */
+    fun play(){
```

Comment:
I'm just wondering when this would be called. The current position in the animation can be controlled with `pause/resume`. `isAnimating` is always true when the animation is active. So the only way this is actually run is when the animation is fully stopped, meaning it matches the implementation of `start`. Does that make sense?

Reply:
let's say there is an AnimationChannel of wall clock starts from 0 to 12 clock
in the app, we want the clock animation starts from current time
so it would be nice to have
```kotlin
setTimeTo(currentTime)
play()
```
and later we may have
```kotlin
play(currentTime)//starts from currentTime
//or to more specific
playFrom(currentTime)
```
to replace the pure start() method
and later we may need playTo(specificPauseTime) method
and may have fluent method like
```java
animation.setTimeTo(currentTime)
                .playTo(specificTime) //when it comes to specific time, it will pause
                .playReverseTo(anotherTimeEarlier) //like resume to another time
                .play(); //continue playing the rest animation
```